test/cbmc/proofs/Makefile-template-defines

```diff
@@ -0,0 +1,19 @@
+
+# Absolute path to the root of the source tree.
+#
+SRCDIR ?= $(abspath $(PROOF_ROOT)/../../..)
+
+
+# Absolute path to the litani script.
+#
+LITANI ?= $(abspath $(PROOF_ROOT)/../litani/litani)
+
+
+# Name of this proof project, displayed in proof reports. For example,
+# "s2n" or "Amazon FreeRTOS". For projects with multiple proof roots,
+# this may be overridden on the command-line to Make, for example
+#
+# 	  make PROJECT_NAME="FreeRTOS MQTT" report
+#
+PROJECT_NAME = "FreeRTOS OTA"
```

Comment:
Where did ou get this project name from? The full name of the library is:
"AWS IoT Over-the-air Update Library". We may want to change it to this.

Reply:
When I was working on the initial proofs, I might have added it without asking anyone. I have changed the PROJECT_NAME now